src/components/Nav.vue

```diff
@@ -32,7 +32,6 @@ export default {
        background: #118AB2;
        height: 100vh;
        width: 22%;
-       position: sticky;
        display: flex;
        flex-flow: column nowrap;
```

Comment:
nowrap is default.
flex-direction: column;  ---> is all you need

Reply:
I think we did it this way for when we eventually add a breakpoint for mobile. That way, we can easily switch it to a header at the top of the page just by changing the flex-flow property. But happy to change it to flex-direction if you think that's better!